modular_skyrat/modules/ashwalkers/code/species/elder.dm

```diff
@@ -0,0 +1,35 @@
+/**
+ * Rewarding Ashwalkers for lasting long enough in the round
+ * 30 minutes: speed
+ * 60 minutes: armor
+ * 90 minutes: fire breath
+ */
+
+/datum/movespeed_modifier/elder_speed
+	multiplicative_slowdown = -0.2
+
+/datum/component/elder_ash
+	///the parent of the component must be a human
+	var/mob/living/carbon/human/human_parent
+
+/datum/component/elder_ash/Initialize()
+	if(!ishuman(parent))
+		return COMPONENT_INCOMPATIBLE
+	human_parent = parent
+	addtimer(CALLBACK(src, .proc/elder_speed), 30 MINUTES)
+	addtimer(CALLBACK(src, .proc/elder_armor), 60 MINUTES)
+	addtimer(CALLBACK(src, .proc/elder_breath), 90 MINUTES)
+
+/datum/component/elder_ash/proc/elder_speed()
+	human_parent.add_movespeed_modifier(/datum/movespeed_modifier/elder_speed)
+	to_chat(human_parent, span_warning("As time passes, you feel that your legs have become stronger-- your walking is more efficient."))
+
+/datum/component/elder_ash/proc/elder_armor()
+	human_parent.dna.species.armor = 30
```

Comment:
I feel like a 30% damage reduction has some chances of leading to cases where they can stack armor and get to 100% armor.

Also, is that just on every armor stat, or is that just for brute/burn, or...?
```suggestion
	human_parent.dna.species.armor = 20
```